vss-extension.json

```diff
@@ -1,7 +1,7 @@
 {
     "manifestVersion": 1,
     "id": "aws-vsts-tools",
-    "version": "1.1.8",
+    "version": "1.1.9",
```

Comment:
This version is not updated manually, please revert

Reply:
reverted version change